src/pybind/mgr/mgr_module.py

```diff
@@ -607,11 +642,37 @@ def __init__(self, module_name, py_modules_ptr, this_ptr):
                     self.MODULE_OPTION_DEFAULTS[o['name']] = str(o['default'])
 
     def __del__(self):
-        unconfigure_logger()
+        self._unconfigure_logging()
 
     @classmethod
-    def _register_commands(cls):
+    def _register_commands(cls, module_name):
+        cls.MODULE_OPTIONS.append(Option(name='log_level', type='str', default="", runtime=True))
+        cls.MODULE_OPTIONS.append(Option(name='log_to_file', type='bool', default=False, runtime=True))
+
         cls.COMMANDS.extend(CLICommand.dump_cmd_list())
+        cls.COMMANDS.extend([{
+            'cmd': '{} logging level set '
```

Comment:
This is similar to `ceph config set mgr debug_mgr 4/4`. What about calling this `{} debug-log level set`. This has no impact on other log channels like `audit`

Reply:
I would prefer to maintain as it is now. Calling it `debug-log` does not sound right, as this is a general log, and I believe that people will not confuse the mgr module log with the ceph audit log.

Reply:
> people will not confuse the mgr module log with the ceph audit log.

I did!